### ISSUE TITLE
Mounting holes M2.5 to M3

### ISSUE THREAD

**Issue by jalinei** [owntech-foundation/TWIST#5]
M2.5 is not a widely used screw standard. M3 would greatly help processes for heatsink and mechanics in general.

**Comment by guigur** [comment]
Agreed, but let's keep the M2.5 for the face plate. M2.5 is the standard for these.

**Comment by jalinei** [comment]
Implemented in v1.4.3, changed the hole diameter while maintaining the same pad size.
